administration/src/cards/CardBlueprint.ts

```diff
@@ -42,10 +43,25 @@ export class CardBlueprint {
       // Extensions valid
       this.extensionHolders.every(state => state.extension.isValid(state.state)) &&
       // Expiration date valid
-      (this.isExpirationDateValid() || this.hasInfiniteLifetime())
+      (this.isExpirationDateValid() || this.hasInfiniteLifetime()) &&
+      // Number of bytes is valid
+      this.hasValidSize()
     )
   }
 
+  hasValidSize(): boolean {
+    // every four characters in a base64 encoded string correspond to three bytes
+    // the qr code can hold 224 characters
+    // therefore 224 / 4 * 3 = 168
```

Comment:
```suggestion
    // every four characters in a base64 encoded string correspond to three bytes
    // the qr code can hold 224 characters
    // therefore 224 / 4 * 3 = 168
    // See https://github.com/digitalfabrik/entitlementcard/issues/690  for more context.
``` 

It would be nice, if you could additionally link the issue https://github.com/digitalfabrik/entitlementcard/issues/690 in this comment to give even more context :)